nx.json

```diff
@@ -103,7 +103,8 @@
       "!{projectRoot}/.eslintrc.json",
       "!{projectRoot}/**/?(*.)+(spec|test).[jt]s?(x)?(.snap)",
       "!{projectRoot}/tsconfig.spec.json",
-      "!{projectRoot}/jest.config.[jt]s"
+      "!{projectRoot}/jest.config.[jt]s",
+      "!{projectRoot}/src/test-setup.[jt]s"
```

Comment:
@lantua this was generated from the migration script, not sure if this changes any testing setup we had existing

Reply:
Should be fine. At worst, it'll add more files to bundled `dist`. Plus we don't have a file matching this pattern, so it should be benign.